backend/protos/mgmt/v1alpha1/anonymization.proto

```diff
@@ -27,8 +33,15 @@ message AnonymizeManyResponse {
   repeated AnonymizeManyErrors errors = 2;
 }
 
+message TransformerMapping {
+  // JQ Expression or Field Path to apply the transformation to
+  string expression = 1;
```

Comment:
Should require that it is min length of 1. `(buf.validate.field).string.min_len = 1`

Reply:
I see in the docs that there is also `[(buf.validate.field).required = true];` which according to the docs it says:
`// - proto3 scalar fields must be non-zero to be considered populated`
Idk if this has always been a thing or maybe at some point it didnt work which is why we use the min_len, but the .required seems a lot more descriptive.